### PR TITLE
FEATURE: RAIL 1036 update examples for the new table component

### PR DESCRIPTION
**Pull request by SeanHochman**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1430/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 6s
- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1549/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 7s
- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED

**Comment by tomasreichmann** [comment]
ok to test

**Comment by tomasreichmann** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1551/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 6s
- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED

**Comment by tomasreichmann** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1552/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1036/ : SUCCESS in 2m 05s

File: examples/src/routes/PivotTableDemo.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            for={() => (
+                <PivotTableSortingExample />
+            )}
+            source={PivotTableSortingExampleSRC}
+        />
+
+        <hr className="separator" />
+
+        <h2 id="measures-row-attributes-and-column-attributes">
+            Example of Presorted Pivot Table by Total
```
Comment (tomasreichmann) [inline examples/src/routes/PivotTableDemo.jsx]: Please update to "Example of Presorted Pivot Table with Totals"

**Comment by tomasreichmann** [comment]
LGTM

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1553/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 9s
- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED

**Comment by tomasreichmann** [comment]
ok to test

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/342/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1038/ : SUCCESS in 2m 34s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/601/ : SUCCESS in 10m 14s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/255/ : SUCCESS in 12m 54s

**Pull request merged**